ai-player.js

```diff
@@ -512,12 +521,12 @@ function createAiPlayer(game, options) {
         // We are in reveal state.
```

Comment:
I think maybe this whole function can be deleted - you are updating `calledBluffs` in `onHistoryEvent` instead, and that should be enough. Do you agree?